### ISSUE TITLE
cell_values drop list if complete path to cell

### ISSUE THREAD

**Issue by waddella** [Roche/rtables#151]
@gmbecker, for the code

```r
library(dplyr)

l <- basic_table() %>% split_cols_by("ARM") %>%
  split_cols_by("SEX") %>%
  split_rows_by("RACE") %>%
  summarize_row_groups() %>%
  split_rows_by("STRATA1") %>%
  analyze("AGE")


tbl <- build_table(l, DM %>% mutate(SEX = droplevels(SEX), RACE = droplevels(RACE)))

row_paths_summary(tbl)
col_paths_summary(tbl)

cell_values(tbl, c("RACE", "ASIAN", "STRATA1", "B"), c("ARM", "A: Drug X", "SEX", "F"))
```

the row and col paths are full paths to a cell however the return value is a list instead of only the nubmer.

I think that if both row and col paths are such that they determine a single cell we should drop the list, do you agree? If so, would you mind adding that to cell_values?

**Comment by gmbecker** [comment]
@wadella this is trivial to implement but I want to discuss it first. It means that the return class will depend on how many rows/columns you're selecting, which generally is enough of an antipattern that we should be careful banking on it. I'm still leaning towards yes, but let's discuss it.

**Comment by gmbecker** [comment]
`value_at` implemented in 1db83eb
